pkg/cache/queue/manager.go

```diff
@@ -88,6 +90,13 @@ func WithResourceTransformations(transforms []config.ResourceTransformation) Opt
 	}
 }
 
+// WithSecondPassBackoff overrides the exponential backoff used for second-pass requeues.
+func WithSecondPassBackoff(b wait.Backoff) Option {
```

Comment:
Is this used anywhere besides unit tests? Will it work in e2e tests?

Reply:
Have thought about it as well, but unsure where to put it? As it's own `SchedulerSecondPassBackoff` in the config, or for it to be related to some existing configuration field?